package.json

```diff
@@ -1,7 +1,7 @@
 {
   "author": "Tom de Grunt <tom@degrunt.nl>",
   "name": "jsonschema",
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Pull requests don't need to touch this file, we can do that at the appropriate time.